website/project/model.py

```diff
@@ -891,7 +891,7 @@ class Node(GuidStoredObject, AddonModelMixin, IdentifierMixin, Commentable):
     # Preprint fields
     preprint_file = fields.ForeignField('StoredFileNode')
     preprint_created = fields.DateTimeField()
-    preprint_provider = fields.StringField()
+    preprint_provider = fields.ForeignField('PreprintProvider')
```

Comment:
As you mentioned earlier, nodes should be able to have more than one affiliated `PreprintProvider`, so this `ForeignField` should be passed a `list=True` kwarg